Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Degree.lean

```diff
@@ -11,20 +11,32 @@ import Mathlib.Tactic.ComputeDegree
 
 This file computes the leading terms of certain polynomials associated to division polynomials of
 Weierstrass curves defined in `Mathlib.AlgebraicGeometry.EllipticCurve.DivisionPolynomial.Basic`.
-In particular, they can be computed by strong induction to be:
+
+## Mathematical background
+
+Let $W$ be a Weierstrass curve over a commutative ring $R$. By strong induction,
  * $\tilde{\Psi}_n = \tfrac{n}{2}X^{\tfrac{n^2 - 4}{2}} + \dots$ if $n$ is even,
  * $\tilde{\Psi}_n = nX^{\tfrac{n^2 - 1}{2}} + \dots$ if $n$ is odd,
  * $\Psi_n^{[2]} = n^2X^{n^2 - 1} + \dots$, and
  * $\Phi_n = X^{n^2} + \dots$.
 
+In particular, when $R$ is an integral domain of characteristic different from $n$, the univariate
```

Comment:
this should say "not dividing n" right?

Reply:
Yes... this got merged a bit too quickly, but I'll make the change in a different PR.